packages/webdriverio/src/utils.js

```diff
@@ -370,3 +370,26 @@ export async function findElements(selector) {
 
     throw INVALID_SELECTOR_ERROR
 }
+
+/**
+ * Strip element object and return w3c and jsonwp compatible keys
+ */
+
+export function verifyArgsAndStripIfElement(args) {
+    function verify(arg) {
+        if (arg.constructor.name === 'Element') {
+            if (!arg.elementId) {
+              throw new Error('property "elementId" missing')
```

Comment:
wrong indentation (add 2 spaces), also let's add a more meaningful message like `The element with selector "${arg.selector}" you trying to pass into the execute method wasn't found`.